includes/class-lookup.php

```diff
@@ -103,6 +105,46 @@ public static function get_preservable_querystring_params_from_url( $url ) {
 		return array_intersect_key( $url_params, array_flip( $preservable_param_keys ) );
 	}
 
+		/**
+		 * Get redirect data status and url based on the provided url.
+		 *
+		 * To make the redirection match, we take a full URL as $url parameter, decode it and keep
```

Comment:
Avoid "we" in documenting:

```suggestion
		 * To make the redirection match, it takes a full URL as $url parameter, decode it and keep
```

Reply:
Likewise for other instances in this PR.